src/redux/city/city-helper.js

```diff
@@ -0,0 +1,9 @@
+const citiesURL = 'https://hotelzilla-api.herokuapp.com/api/cities';
+
+const getAllCities = async () => {
+  const response = await fetch(citiesURL);
+  const data = await response.json();
+  return data;
+};
+
+export default getAllCities;
```

Comment:
- [ ] [Optional] I strongly recommend you to use `createAsyncThunk` function that comes with redux toolkit. It will give you more options to handle server response